### PR TITLE
update license limits

### PR DESCRIPTION
**Pull request by jdeoliveira**

### CODE REVIEW

File: modules/ROOT/pages/licensing-overview.adoc [hunk 1]
```diff
@@ -10,9 +10,19 @@ Anypoint API Community Manager licenses are procured and provisioned by MuleSoft
 . Five (5) admin users (community administrators or operators)
 . Connectivity to one (1) Anypoint Platform top-level organization
 . Up to 10,000 member accounts across all communities
+. Salesforce Standard Objects, Features, Capability, and Custom Objects in Customer Community Plus 
+. One (1) megabyte storage per user for all communities
+. 10 megabyte storage for organization and session caches
+. 1,000,000 page views per month across all communities
```
Comment (kbostrom) [inline modules/ROOT/pages/licensing-overview.adoc]: "One (1) million"?

File: modules/ROOT/pages/licensing-overview.adoc [hunk 2]
```diff
@@ -10,9 +10,19 @@ Anypoint API Community Manager licenses are procured and provisioned by MuleSoft
...
+. 40 gigabit network bandwidth in production for all communities and 1 gigabit network bandwidth for all sandboxes
```
Comment (kbostrom) [inline modules/ROOT/pages/licensing-overview.adoc]: Add "environments" after production and sandbox (es) ?

File: modules/ROOT/pages/licensing-overview.adoc [hunk 3]
```diff
@@ -10,9 +10,19 @@ Anypoint API Community Manager licenses are procured and provisioned by MuleSoft
...
 
 Community login-based license is calculated by the number of *daily unique logins*: login actions that different members perform daily in your communities during a month. Members associated with your license consume one login each day they log into any of your communities. Logging in multiple times during the same day only consumes one login. Once logged in, switching between communities doesn’t consume extra logins.
 
 Forecasting the adequate number of daily unique logins is critical to ensure that you have enough licenses to satisfy the demand your community will generate. Your MuleSoft account team will assist you in this analysis as part of the capacity planning and community design process before you roll out your community.
 
 You can find more information about *Salesforce Customer Community Plus Login License Unlimited Edition* in the Salesforce Community Cloud documentation page https://help.salesforce.com/articleView?id=users_license_types_communities.htm&type=5[Communities User Licenses, role=external, window=blank].
+
+Additional capacity can be provisioned to your organization if required. For more information, please reach out to your MuleSoft representative.
```
Comment (kbostrom) [inline modules/ROOT/pages/licensing-overview.adoc]:
```suggestion
Additional capacity can be provisioned to your organization if required. For more information, contact your MuleSoft representative.
```

**Review by kbostrom — APPROVED** [review]
Here you go!

File: modules/ROOT/pages/licensing-overview.adoc (hunk 1 above)
Comment (jdeoliveira) [reply modules/ROOT/pages/licensing-overview.adoc]: I like "One (1) million"

File: modules/ROOT/pages/licensing-overview.adoc (hunk 2 above)
Comment (jdeoliveira) [reply modules/ROOT/pages/licensing-overview.adoc]: yes - agree

File: modules/ROOT/pages/licensing-overview.adoc (hunk 3 above)
Comment (jdeoliveira) [reply modules/ROOT/pages/licensing-overview.adoc]: agree

File: modules/ROOT/pages/licensing-overview.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/licensing-overview.adoc]: Much better! Thank you.

File: modules/ROOT/pages/licensing-overview.adoc (hunk 2 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/licensing-overview.adoc]: Also much better! Thank you.

**Review by wwarner-salesforce — APPROVED** [review]

**Pull request merged**